### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by XhmikosR**
https://github.com/mdo/github-buttons/pull/79/files?w=1

### CODE REVIEW

**Comment by mdo** [comment]
Whoa, awesome. I'll test this all out and they push it soon. Thanks!

**Comment by XhmikosR** [comment]
Added one more patch. Saved a few bytes :)

**Comment by XhmikosR** [comment]
Added one more patch so that the HTML code is valid. I personally don't like inline styles, but I'm not sure how else to fix this. :/

**Comment by XhmikosR** [comment]
Any news @mdo? Do you need me to make any further changes?

**Comment by zenorocha** [comment]
I guess it's better to send these commits in separated PRs @XhmikosR, that way it gets easier to review.

**Comment by mdo** [comment]
I can jump back on this soon and resolve the conflicts. Totally just fell to the wayside with Bootstrap and GitHub stuff :).

File: _layouts/default.html [hunk 1]
```diff
@@ -2,32 +2,35 @@
...
       <div class="tweet-button">
-        <a href="https://twitter.com/share" class="twitter-share-button" data-count="none" data-via="mdo">Tweet</a><script type="text/javascript" src="//platform.twitter.com/widgets.js"></script>
+        <a href="https://twitter.com/share" class="twitter-share-button" data-count="none" data-via="mdo">Tweet</a>
       </div>
 
       <p>Introducing the unofficial</p>
       <h1>GitHub buttons</h1>
       <p>Showcase your GitHub repo&#39;s success with hotlinkable GitHub star, fork, or follow buttons. Available in two sizes with up-to-date counts.</p>
-      <p>Made by <a href="http://twitter.com/mdo">@mdo</a>. Available on <a href="https://github.com/mdo/github-buttons">GitHub</a>. Licensed <a href="http://www.apache.org/licenses/LICENSE-2.0&quot;">Apache 2</a>.</p>
+      <p>Made by <a href="https://twitter.com/mdo">@mdo</a>. Available on <a href="https://github.com/mdo/github-buttons">GitHub</a>. Licensed <a href="http://www.apache.org/licenses/LICENSE-2.0">Apache 2</a>.</p>
```
Comment (mdo) [inline _layouts/default.html]: Making a note for myself—this is probably the line that conflicts.

**Comment by XhmikosR** [comment]
It's not a matter of separating the patches, rather no time.

@mdo; I'll rebase now.

**Comment by XhmikosR** [comment]
Weird, my local git just rebased this without any conflicts... Something seems fishy with the way the conflicts are detected on GH.

Regardless, repushed.

File: _layouts/default.html (hunk 1 above)
Comment (XhmikosR) [inline _layouts/default.html]: Nah, it was the 404 error fix in the validation errors fixes patch :)

File: page.css [hunk 2]
```diff
@@ -1,5 +1,4 @@
 ---
-layout: null
```
Comment (Walkman100) [inline page.css]: Does this still work?? If so, very cool. (I thought the layout tag was required)

Comment (XhmikosR) [inline page.css]: Works for months now.

Comment (Walkman100) [inline page.css]: I meant is the css file still rendered with jekyll...

Comment (XhmikosR) [inline page.css]: Yeah, that is the whole point.

Comment (Walkman100) [inline page.css]: Ok, thanks! :smile:

**Pull request merged**